src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
+  public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk {
+    protected CodeAccessPermission() { }
+    public void Assert() { }
+    public abstract System.Security.IPermission Copy();
+    public void Demand() { }
+    public void Deny() { }
+    public override bool Equals(object obj) { return default(bool); }
+//    public abstract void FromXml(System.Security.SecurityElement elem);
```

Comment:
I guess SecurityElement needs to be added to System.Security.Principal (as opposed to System.Security.Permissions, this one). I think Krzysztof put it there, a layer below, because it's used by his System.Security.Cryptography.Algorithms.dll, and he didn't want that to depend on all of CAS.
I think it would be a no-op implementation, like the rest of CAS.
It seems okay to me to open an issue if you can't do it immediately (since this change is large enough) as long as you do it directly afte.r 

---
In reply to: [70840201](https://github.com/dotnet/corefx/pull/10049#discussion_r70840201) [](ancestors = 70840201,70814453)